templates/themes/1/home.amp.html

```diff
@@ -0,0 +1,109 @@
+<!---
+Copyright 2017 The AMPStart Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!doctype html>
+{{#blog-photogallery}}
+<html ⚡>
+{{> ../utils/amp-page-head.snip.html}}
+<body>
+  {{> ../components/navbar/navbar.snip.html}}
+
+  {{#fullpage-hero}}
+    <!-- Start Fullpage Hero -->
+    <figure class="ampstart-image-fullpage-hero m0 relative">
+      <amp-img width="404" height="720" layout="responsive" src="{{narrow-img}}" media="(max-width: 415px)"></amp-img>
```

Comment:
We need to use treble braces for attribute value variables when they’ll include paths with slashes, so `{{{narrow-img}}}` here and throughout this file for `src`, `href` and any others.